src/App/Module.php

```diff
@@ -80,15 +81,10 @@ class Module
 	private $module;
 
 	/**
-	 * @var BaseModule The module class
+	 * @var ICanHandleRequests The module class
 	 */
 	private $module_class;
```

Comment:
```suggestion
	 * @var ICanHandleRequests The module object
	 */
	private $module;
```
It sounds redundant with this class name, can we think about a different terminology for the `App\Module` class and the `ICanHandleRequests` classes? Like `Module` and `Controller` or something?